Changelog.md

```diff
@@ -1,5 +1,9 @@
 # FOSSA CLI Changelog
 
+## 3.9.27
+
+- Tar: Move to upstream tar rather than a custom fork. ([#1452](https://github.com/fossas/fossa-cli/pull/1452))
```

Comment:
Since this will release on its own, please consider adding a bit of an explanation of the effect for end users.

For example, "FOSSA CLI will more reliably unpack containers" etc.

Reply:
Thanks, I've added a little more detail.